cpp/libjsonnet++.cc

```diff
@@ -0,0 +1,157 @@
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
I noticed it did not include many new headers, so if it (libjsonnet++) were a header only library, it would not need a new build target and would be much simpler.  No need to do it in this PR but it's worth considering.